frontend/device/cervantes/powerd.lua

```diff
@@ -51,10 +51,13 @@ function CervantesPowerD:init()
         if self.device:hasNaturalLight() then
             local nl_config = G_reader_settings:readSetting("natural_light_config")
             if nl_config then
-                for key,val in pairs(nl_config) do
+                for key, val in pairs(nl_config) do
                     self.device.frontlight_settings[key] = val
                 end
             end
+            -- Does this device's NaturalLight use a custom scale?
```

Comment:
Not sure anymore :p

Here you are: https://github.com/koreader/koreader/issues/4291#issuecomment-439700537

I hope it helps :)

Reply:
Yup, I'm familiar with that PWM controller, it's on quite a few NTX boards ;).